micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/MetricsAspect.groovy

```diff
@@ -56,5 +66,23 @@ class MetricsAspect {
                 trimLeadingCharacter(str, '.' as Character), '.' as Character)
     }
 
+    public static String sanatizeIpAddressIfPresent(String metricName) {
```

Comment:
InetAddressValidator is definitely more elegant in asserting if the string is a IP address or not but it can't do replacing of the dots with the underscores. I'll still have to use the regex Matcher for that.